dev-tools/resolve_translation_conflicts.sh

```diff
@@ -9,20 +9,32 @@ source "$(dirname "${BASH_SOURCE[0]}")/_functions.sh"
 require_installed
 ensure_not_root
 
-cd "${BASE_DIR}/src/cms/locale/de/LC_MESSAGES" || exit 1
+# Relative path to translation file from the base directory
+TRANSLATION_FILE="src/cms/locale/de/LC_MESSAGES/django.po"
 
-# Replace git conflict markers
-sed -i -E -e 's/<<<<<<< HEAD//g' django.po
-sed -i -E -e 's/=======//g' django.po
-sed -i -E -e 's/>>>>>>> .+//g' django.po
+echo "Resolving translation conflicts..." | print_info
 
-# Remove duplicated translations
-msguniq -o django.po django.po
+# Remove git conflict markers
+sed --in-place --regexp-extended -e '/^<<<<<<< .+$/d' -e '/^=======$/d' -e '/^>>>>>>> .+$/d' "$TRANSLATION_FILE"
+
+# Remove duplicated translations and show error if this is not possible
+if ! msguniq --output-file "$TRANSLATION_FILE" "$TRANSLATION_FILE" 2>&1 \
+  | sed --regexp-extended -e "s|^(${TRANSLATION_FILE}):([0-9]+):[0-9]+?:? (.+)$|\1 \x1b[1m(line \2) \x1b[1;31m\3\x1b[0;39m|g" -e "$ d"; then
+    echo -e "\n❌ Not all conflicts could be solved automatically" | print_error
+    echo "Please fix the mentioned problem(s) manually and run this script again." | print_bold
+    exit 1
+fi
 
 # Check if conflicts remain
-if grep -q "#-#-#-#-#" django.po; then
-    echo "Not all conflicts could be solved automatically. Please resolve remaining conflicts manually (marked with \"#-#-#-#-#\")."
-else
-    # Fix line numbers and empty lines
-    bash "${DEV_TOOL_DIR}/translate.sh"
+if grep --quiet "#-#-#-#-#" "$TRANSLATION_FILE"; then
+    echo -e "\n$TRANSLATION_FILE"
+    grep --before-context=2 --after-context=1 --line-number "#-#-#-#-#" "$TRANSLATION_FILE" | format_grep_output | print_with_borders
+    echo -e "❌ Not all conflicts could be solved automatically" | print_error
```

Comment:
Just a small thing: Do you want to add the "\n" as in line 23?

Reply:
Since the function [print_with_borders](https://github.com/Integreat/integreat-cms/blob/4882518cef39864502051860d6b17b1d7707d284/dev-tools/_functions.sh#L88) already outputs one additional newline at its end, this is not required here. In both cases, there is an empty new line between the previous line and the error message. You can try this out be recreating both error messages manually (a syntax error is pretty easy by just breaking something, and a `#-#-#-#-#` conflict can be created by inserting two different translation for the same source string).

Reply:
Ahh I see :)